index.html

```diff
@@ -1,17 +1,63 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>My Portfolio</title>
-    <link href="style.css">
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <nav>
```

Comment:
- [ ] Let's make sure that we use a `<header>` tag instead of `<nav>` because semantically, the navigation should be located within the header section 😄

Reply:
Issue not resolved